### PR TITLE
Re-Integrate Knative build v0.4.0

### PR DESCRIPTION
**Pull request by JoergBastian**
**Description**

Knative build was part of the Kyma Knative installation until PR #3051 .
Proposed is the re-integration of the build component.

Changes proposed in this pull request:

- Add `knative-build` helm chart

### CODE REVIEW

File: resources/knative-build/values.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+global:
+  isLocalEnv: false
+  knative: false
```
Comment (pbochynski) [inline resources/knative-build/values.yaml]: It should be enabled.

> 
```suggestion
  knative: true
```

**Comment by k15r** [comment]
This issue is outdated

**Comment by k15r** [comment]
Outdated, closing

**Pull request closed without merging**